src/Animal.js

```diff
@@ -1,12 +1,113 @@
-import { Organism } from './organism.js';
+import { Organism } from './Organism';
+import { findNearestTiles } from './findNearestTiles';
+import { classesList } from './classesList';
+import { findRandomTileInArray } from './findRandomTileInArray';
 
 export class Animal extends Organism {
-    constructor() {
-        super();
-        this.strength = 0;
+  constructor(board) {
+    super(board);
+    this.strength = 0;
+  }
+
+  mate(newTile, organism) {
+    if (newTile.currentOrganism) {
```

Comment:
to avoid the `if` statements nesting you can perform an early-exit.
```suggestion
    if (!newTile.currentOrganism) {
      return false; 
    }
```
this `false` is only for the purpose of consistent return type, because you are returning a `true` value if the mating was success. It is a good practice to keep the return type consistency